library/farmosUtil/farmosUtil.checkPlantAsset.unit.cy.js

```diff
@@ -0,0 +1,160 @@
+import * as farmosUtil from './farmosUtil.js';
+
+describe('Test the bed utility functions', () => {
+  let bedMap = null;
+  let fieldMap = null;
+  let greenHouseMap = null;
+
+  before(() => {
+    cy.wrap(farmosUtil.getBedNameToAssetMap()).then((map) => {
+      bedMap = map;
+    });
+    cy.wrap(farmosUtil.getFieldNameToAssetMap()).then((map) => {
+      fieldMap = map;
+    });
+    cy.wrap(farmosUtil.getGreenhouseNameToAssetMap()).then((map) => {
+      greenHouseMap = map;
+    });
+  });
+
+  beforeEach(() => {
+    cy.restoreLocalStorage();
+    cy.restoreSessionStorage();
+  });
+
+  afterEach(() => {
+    cy.saveLocalStorage();
+    cy.saveSessionStorage();
+  });
+
+  it('Check that returned plant assets are active', () => {
+    const locationName = 'ALF';
+    const checkedBeds = ['ALF-1', 'ALF-2'];
+
+    cy.wrap(farmosUtil.getPlantAssets(locationName, checkedBeds)).then(
+      (plantAssetIds) => {
```

Comment:
Note that this test will pass if the call to `getPlantAssets` fails and returns an empty array.  A similar thing happens in the other tests as well.  Maybe add a check that the result is not empty, and then check that all the assets are active?